src/icons/service-serverless-gcp.svg

```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 23 32" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="m1.87 32 8.225-7.405v-1.871l-6.98 6.308v-2.908l15.013-13.649v2.93l-1.495 1.332 1.184.747 1.597-1.425v-5.663l-3.497.486 3.135-5.274V.632L18.357 0 0 15.895v3.89l5.89-.999-4.022 6.836V32h.001zM17.767 3.172V5.22l-3.365 5.87-1.713.239 5.078-8.158zM1.234 18.23v-2.101l7.167-.982-1.505 2.18-5.662.903z" fill="#FD5750"/><g clip-path="url(#a)"><path d="M18.614 20.66h.367l1.043-1.044.051-.443a4.689 4.689 0 0 0-7.627 2.287.568.568 0 0 1 .366-.022l2.086-.344s.106-.175.16-.164a2.602 2.602 0 0 1 3.562-.27h-.008z" fill="#EA4335"/><path d="M21.51 21.46a4.698 4.698 0 0 0-1.417-2.283L18.63 20.64c.618.505.97 1.267.955 2.065v.26a1.303 1.303 0 1 1 0 2.605h-2.606l-.26.264v1.563l.26.26h2.606a3.39 3.39 0 0 0 1.925-6.197z" fill="#4285F4"/><path d="M14.37 27.642h2.605v-2.086H14.37c-.185 0-.369-.04-.538-.117l-.366.113-1.05 1.043-.092.367a3.373 3.373 0 0 0 2.046.68z" fill="#34A853"/><path d="M14.37 20.875a3.389 3.389 0 0 0-2.046 6.075l1.511-1.511a1.303 1.303 0 1 1 1.724-1.724l1.511-1.512a3.388 3.388 0 0 0-2.7-1.328z" fill="#FBBC05"/></g><defs><clipPath id="a"><path fill="#fff" transform="translate(11 18)" d="M0 0h12v9.657H0z"/></clipPath></defs></svg>
```

Comment:
These are general change we make with svgs.
remove fill="none" in svg tag
change fill="currentColor" in path tag
Please make the change unless otherwise required

Reply:
There are multiple colors in this one, so can't do it.